From f0108782e283725d7b6588745edeec06a8a379e5 Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Thu, 14 Jan 2010 09:30:56 +0100
Subject: [PATCH 2/2] install tools

Add make targets to install the tools.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
[mk: fix to work with old coreutils]
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
---
 Makefile.opts.in |    1 +
 src/Makefile     |    6 ++++--
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/Makefile.opts.in b/Makefile.opts.in
index 87d229b..93467c8 100644
--- a/Makefile.opts.in
+++ b/Makefile.opts.in
@@ -21,6 +21,7 @@ LIBNL_LIB        := @LIBNL_LIB@
 prefix           := @prefix@
 exec_prefix      := @exec_prefix@
 libdir           := @libdir@
+sbindir          := @sbindir@
 includedir       := @includedir@
 mandir           := @mandir@
 sysconfdir       := @sysconfdir@
diff --git a/src/Makefile b/src/Makefile
index ddbe29a..941bb10 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -39,7 +39,9 @@ clean:
 
 distclean: clean
 
-install:
-	@true
+install: $(TOOLS:=-install)
+
+%-install: %
+	install -m755 $< $(DESTDIR)$(sbindir)/
 
 include ../Makefile.rules
-- 
1.6.6

